### DISCUSSION TITLE
What kind of posts are planned for this discussion?

### DISCUSSION THREAD

**Discussion by yuki-kimoto** [mojolicious/mojo#1639] [Q&A] [answered]
What kind of posts are planned for this discussion?

Is it for core team discussions?

Is it possible for individual or corporate users using Mojolicious to have a conversation?

**Comment by kraih** [comment, marked as answer]
This is an open forum, anyone can ask questions related to Mojolicious. And they may or may not receive an answer. There are no planned discussions.

**Reply by yuki-kimoto** [reply]
OK. 

Gihub Disscussion feature is used, but this is, in fact,  open forum.

I misunderstood. others may misunderstand.

Anyway, this is good news!!!
